action/login.js

```diff
@@ -0,0 +1,28 @@
+export default async function login(email, password) {
+  const url = 'https://bootcamp-api.codeit.kr/api/sign-in'
+  const data = {
+    email,
+    password,
+  }
+
+  const options = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(data),
+  }
+
+  try {
+    const response = await fetch(url, options)
+    const responseData = await response.json()
```

Comment:
@ugaugaugaugaugauga 

payload를 따로 받기보단, 전달받는곳에서 dto로 한번에 전달받아보면 어떤가요? 

```suggestion
export default async function login(loginOption) {
  try {
    const response = await fetch(`${BASE_API_ENDPOINT}sign-in`, {
      method: 'POST',
      headers: {
        'Content-Type': 'application/json',
      },
      body: JSON.stringify(loginOption),
  }

```

Reply:
한번에 처리하는게 더 직관적이내요!